server_installation/topics/operator/external-database.adoc

```diff
@@ -35,7 +36,22 @@ The other properties work in the same way for a hosted or external database. Set
 * `POSTGRES_DATABASE` - Database name to be used.
 * `POSTGRES_USERNAME` - Database username
 * `POSTGRES_PASSWORD` - Database password
-* `POSTGRES_SUPERUSER` - Indicates, whether backups should run as super user. Typically `true`.
+* `POSTGRES_SUPERUSER` - Indicates whether backups should run as super user. Typically `true`.
+* `SSL_MODE` - Indicates whether to use TLS on the connection to the external PostgreSQL database. Please check the possible https://www.postgresql.org/docs/current/libpq-ssl.html[values]
```

Comment:
No need to use "please" in technical documentation.

Reply:
:+1: